docs/ar/week02/02-3.md

```diff
@@ -0,0 +1,448 @@
+
+
+<!-- ## Spiral classification - Jupyter notebook -->
+## التصنيف اللولبي - Jupyter مذكرة
+
+<!-- The Jupyter notebook can be found [here](https://github.com/Atcold/pytorch-Deep-Learning-Minicourse/blob/master/04-spiral_classification.ipynb). In order to run the notebook, make sure you have `the dl-minicourse` environment installed as specified in [README.md](https://github.com/Atcold/pytorch-Deep-Learning-Minicourse/blob/master/README.md).
+
+An explanation of how to use `torch.device()` can be found in [last week's notes](https://atcold.github.io/pytorch-Deep-Learning-Minicourse/en/week01/01-3/).
+
+Like before, we are going to be working with points in $\mathbb{R}^2$ with three different categorical labels -- in red, yellow and blue -- as can be seen in **Fig. 8**. -->
+
+يمكن العثور على مذكرة Jupyter [هنا](https://github.com/Atcold/pytorch-Deep-Learning-Minicourse/blob/master/04-spiral_classification.ipynb). لتشغيل المذكرة، تأكد من تثبيت بيئة `the dl-minicourse` كما هو موضح في [README.md](https://github.com/Atcold/pytorch-Deep-Learning-Minicourse/blob/master/README.md).
+
+يمكن العثور على شرح لكيفية استخدام `torch.device()` في [ملاحظات الأسبوع الماضي](https://atcold.github.io/pytorch-Deep-Learning-Minicourse/en/week01/01-3/).
+
+كما في السابق، سنعمل مع النقاط في $\mathbb{R}^2$ بثلاث تسميات لفئات مختلفة - باللون الأحمر والأصفر والأزرق - كما هو موضح في **شكل 8**.
+
+<center>
+<img src="{{site.baseurl}}/images/week02/02-3/2-data.png" style="zoom: 50%; background-color:#DCDCDC;" /><br>
+<b>شكل 8</b> بيانات التصنيف اللولبي.
+</center>
+
+<!-- `nn.Sequential()` is a container, which passes modules to the constructor in the order that they are added; `nn.linear()` is miss-named as it applies an **affine** transformation to the incoming data: $\boldsymbol y = \boldsymbol W \boldsymbol x + \boldsymbol b$. For more information, refer to the [PyTorch documentation](https://pytorch.org/docs/stable/nn.html).
+
+Remember, an affine transformation is five things: rotation, reflection, translation, scaling and shearing.
+
+As it can be seen in **Fig. 9**, when trying to separate the spiral data with linear decision boundaries - only using `nn.linear()` modules, without a non-linearity between them - the best we can achieve is an accuracy of $50\%$. -->
+
+`nn.Sequential()` هو عبارة عن حاوية (container)، تمرر الوحدات إلى المُنشئ (constructor) بالترتيب الذي تم إضافتها إليها. مسمى `nn.linear()` هو تسمية غير مطابقة للوظيفة حيث أنه يسخدم لتطبيق تحويل **تآلفي (affine)** على البيانات الواردة: $\boldsymbol y = \boldsymbol W \boldsymbol x + \boldsymbol b$. لمزيد من المعلومات، راجع [PyTorch وثائق](https://pytorch.org/docs/stable/nn.html).
```

Comment:
This looks funny
![image](https://user-images.githubusercontent.com/2119355/123871493-c21ff100-d901-11eb-8ffc-ae4020abb619.png)